### PR TITLE
Remove stray whitespace around scope resolution operator.

### PR DESCRIPTION
**Pull request by Eelis**
diff

### CODE REVIEW

File: source/templates.tex [hunk 1]
```diff
@@ -1483,7 +1483,7 @@
 the
 \grammarterm{template-argument}
 for
-\tcode{Array<T>\,::\,op\-er\-a\-tor\,[]\,()}
+\tcode{Array<T>::op\-er\-a\-tor\,[]\,()}
```
Comment (Eelis) [inline source/templates.tex]: Unrelatedly, this line has an instance of a function being referred to as "f()" even though it doesn't have zero parameters and cannot be called with zero arguments. That practice seems a bit sloppy for the standard (and certainly isn't used consistently in the document). Thoughts?

Comment (jensmaurer) [inline source/templates.tex]: Please open an editorial issue for this (I thought we already had one, but maybe not.)

Comment (jwakely) [inline source/templates.tex]: Yes that's definitely wrong (but probably not the only place we do it).

**Comment by jwakely** [comment]
We aren't (currently) benefiting from adding those hints in those locations, so removing them to get rid of the whitespace seems better.

**Comment by jensmaurer** [comment]
Looks good to me.

**Pull request merged**